torch/nested/__init__.py

```diff
@@ -364,10 +364,8 @@ def nested_tensor_from_jagged(
 
     >>> values = torch.randn(12, 5)
     >>> offsets = torch.tensor([0, 3, 5, 6, 10, 12])
-    >>> nt = nested_tensor_from_jagged(values, offsets)
     >>> # 3D shape with the middle dimension jagged
-    >>> nt.shape
-    torch.Size([5, j2, 5])
```

Comment:
Yeah, the j-numbers changed from j2 to j-something-else.

I can probably add a comment with the shape instead of printing it out, just to show users what the shape looks like.